### ISSUE TITLE
Velden verschoven in excel rapport geodata

### ISSUE THREAD

**Issue by gerardnienhuis** [IDgis/geoportaal-test#463]
Het blijkt dat de csv die ingelezen wordt in excel, niet altijd de juiste waarden in de juiste kolommen zet. Mogelijk komt dit omdat excel anders dan andere spreadprogramma's de csv anders inleest of converteert. Probleem is bijvoorbeeld zichtbaar bij wms_extern, zoals in bijgevoegd figuur te zien is (geel gemarkeerde cellen hadden een waarde moeten bevatten, zijn nu leeg).

Verzoek: een keer uitzoeken en zorgen, eventueel door aanpassing van inhoud van het rapport en/of aanpassing van veldscheiders in het rapport, dat ook in excel deze csv goed wordt ingelezen.

![fout_rapport_verschoven_velden](https://cloud.githubusercontent.com/assets/9005444/25180339/e7eecab6-250d-11e7-9415-baba90ae0d43.jpg)

**Comment by SandroNeumann** [comment]
Als je de csv inleest heb je dan onderstaande gegevens aangegeven? Dus scheidingsteken ;  en tekstscheidingsteken ".

![inlezen xml](https://cloud.githubusercontent.com/assets/10757125/25181015/6316a748-2510-11e7-87dc-4b7717f00501.png)

**Comment by gerardnienhuis** [comment]
Dat soort instellingen krijg je default bij provincie Overijssel. Wens is dat je gewoon op de csv klikt, en dat dan, met de default instellingen, de inhoud goed op het scherm komt. In excel.

**Comment by SandroNeumann** [comment]
Dat kan je vanuit de programmatuur niet afdwingen. Je downloadt een csv en daar doe je vervolgens wat mee in een extern programma.

**Comment by gerardnienhuis** [comment]
klopt, maar het lijkt er op alsof het, door ons gebruikte externe programma (excel) niet helemaal doet, wat je zou verwachten. Wens is, dat alle data in de juiste cellen terecht komt. Dat is nu niet altijd het geval.

**Comment by SandroNeumann** [comment]
Dat is een wens die je alleen bij Overijssel of Excel kunt neerleggen. Het geoportaal levert alleen een csv met bepaalde scheidingstekens, die je vervolgens in een spreadsheet programma moet aan kunnen geven.

**Comment by gerardnienhuis** [comment]
Zou het helpen om de volgorde van de kolommen te veranderen? Of het type veldscheider wat is gebruikt?

**Comment by SandroNeumann** [comment]
Als jij er achter kunt komen wat voor scheidingstekens bij jullie default gebruikt wordt kunnen we dat aanpassen.

**Comment by gerardnienhuis** [comment]
zie ook #445

**Comment by SandroNeumann** [comment]
Zou je de exacte plek in het rapport kunnen aanwijzen waar zich dit voordoet? Dan kunnen we kijken of zich dat in andere spreadsheet programma's ook voordoet. Tevens zou je moeten achterhalen welke scheidingstekens bij Overijssel ingesteld staan.

**Comment by SandroNeumann** [comment]
Eventueel excel output genereren.

**Comment by gerardnienhuis** [comment]
excelfile met voorbeeld is naar @SandroNeumann gemaild.

**Comment by SandroNeumann** [comment]
**Bevindingen**:
In Excel is een record waar in de kolom "abstract" een stuk of 25 puntkomma's staan. Alleen bij de laatste twee splitst deze naar volgende kolommen. In Libre Office is dit probleem niet en blijft het in dezelfde kolom staan.
